config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  root to: 'blogs#index'
```

Comment:
```suggestion
  root 'blogs#index'
```
rootの場合、toはなくても大丈夫です。